### DISCUSSION TITLE
Lesson 9 ERC20: ImportError: cannot import name 'PromptSession'

### DISCUSSION THREAD

**Discussion by almurillo** [smartcontractkit/full-blockchain-solidity-course-py#994] [Q&A] [answered]
Hi All,
I have been having this issue when I run the command 'brownie run scripts/1_deploy_token' I get the following error:
ImportError: cannot import name 'PromptSession' from 'prompt_toolkit'. 
Does anyone else have that issue? if so how do you fix it?
![image](https://user-images.githubusercontent.com/12633384/153066736-62d1fe4a-b1b0-4c61-bd5a-ebe9c44276df.png)

**Comment by cromewar** [comment]
this is most likely due to the python version, I strongly suggest to use a `virtualenv` and python version 3.8

**Reply by almurillo** [reply]
Hey @cromewar thanks for the reply but I am still having the same issue. I had started my virtual environment with pipenv python 3.8.2 and then into shell but I still get the same error. It looks like brownie is looking at the same directory. ImportError: cannot import name 'PromptSession' from 'prompt_toolkit' (/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/prompt_toolkit/__init__.py).
This started when I had to update anaconda jupiter notebooks.

**Reply by cromewar** [reply]
Did you change the python interpreter en vscode? And selected the one with the virtual env ?

**Reply by almurillo** [reply]
yes I did and I keep getting the same results.
![image](https://user-images.githubusercontent.com/12633384/153247946-0c6d78ac-9854-46b3-b8cc-f18cb50079f1.png)

**Comment by almurillo** [comment, marked as answer]
After hours of testing I was able to fix it. If someone has the same problem, you will need to uninstall python completely and install it again as well as brownie and pipenv. That solved my problem.

**Reply by cromewar** [reply]
So it was something related to your python installation then... I'm glad is solved now.

**Reply by almurillo** [reply]
@cromewar It was working before I install Anaconda so I had uninstall Anaconda and Python to get it going. Thanks for your help.
